src/almalinux/8/helix/amd64/Dockerfile

```diff
@@ -34,7 +31,3 @@ ENV LANG=en-US.UTF-8
 RUN adduser --uid 1000 --shell /bin/bash --gid adm helixbot \
     && chmod 755 /root \
     && echo "helixbot ALL=(ALL)       NOPASSWD: ALL" > /etc/sudoers
-
-USER helixbot
-
-RUN python3 -m venv /home/helixbot/.vsts-env
```

Comment:
removing the helix-scripts packages should be fine b/c we mount the host's installation location when starting Helix containers. however it seems like a **Bad Idea :tm:** since a `venv` isolates test/user code from the system Python environment